torchvision/transforms/transforms.py

```diff
@@ -1760,7 +1760,7 @@ def __repr__(self) -> str:
 class GaussianBlur(torch.nn.Module):
     """Blurs image with randomly chosen Gaussian blur.
     If the image is torch Tensor, it is expected
-    to have [..., C, H, W] shape, where ... means an arbitrary number of leading dimensions.
+    to have [N, C, H, W] shape, where ... means an arbitrary number of leading dimensions.
```

Comment:
As is the comment is not correct:

1. Shape `[C, H, W]` is also allowed. A batch dimension is not required.
2. The second part of the sentence references "...", which no longer exists.

You could align the sentence with the one from `F.pad`

https://github.com/pytorch/vision/blob/4af683107f967d4d435be4180020989dd8c3019c/torchvision/transforms/functional.py#L498-L499

e.g.

```suggestion
    to have [..., C, H, W] shape, where ... means at most one leading dimension.
```

Reply:
@pmeier 

Thank you for reviewing! I have changed the text as you suggested.